### DISCUSSION TITLE
How can i zoom video?

### DISCUSSION THREAD

**Discussion by YiDongHyung** [jhomlala/betterplayer#521] [Q&A]
Hi. 
I'm developing CCTV app. Zooming a video during the playback is an important function.
But I don't know how to zoom video using betterplayer.

How can i zoom video?

**Comment by jhomlala** [comment]
There's no feature in Better Player which allows to zoom the video. You may try use aspect ratio and fit to make video bigger. You can also try to use this widget: https://api.flutter.dev/flutter/widgets/InteractiveViewer-class.html, but I'm not sure if it will work.
